akka-stream/src/main/scala/akka/stream/impl/FanoutProcessor.scala

```diff
@@ -145,6 +142,11 @@ import org.reactivestreams.Subscriber
     primaryOutputs.complete()
   }
 
+  override def postStop(): Unit = {
+    super.postStop()
+    timeoutTimer.foreach(_.cancel())
```

Comment:
pattern match instead of foreah avoids another alloc here, probably not super important though